staging/src/k8s.io/metrics/pkg/client/clientset_generated/clientset/typed/metrics/v1alpha1/podmetrics.go

```diff
@@ -80,8 +79,8 @@ func (c *podMetricses) List(opts v1.ListOptions) (result *v1alpha1.PodMetricsLis
 
 // Watch returns a watch.Interface that watches the requested podMetricses.
 func (c *podMetricses) Watch(opts v1.ListOptions) (watch.Interface, error) {
+	opts.Watch = true
 	return c.client.Get().
-		Prefix("watch").
```

Comment:
> a release note and a client release note.

The changes was done in https://github.com/kubernetes/kubernetes/pull/41887 actually. I'll update the client-go's CHANGELOG.

This PR just lets metrics client use the latest client-gen.